src/statement/statement.es6

```diff
@@ -89,6 +90,7 @@ const refreshTable  = (yyy_mm_dd) => {
             .then(refresh)
             .catch((err) => {
                   refresh({});
+                  api_limit_reached = true;
```

Comment:
Should this be reset after one minute? (since the rate limit will be cleared by then)